packages/itwinui-css/backstop/tests/non-ideal-state.html

```diff
@@ -35,7 +35,7 @@
         border-bottom: solid 2px #000;
       }
 
-      .iui-non-ideal-state-scrolling:last-child {
+      .iui-non-ideal-state-scrolling:last-of-type {
```

Comment:
Why is this necessary for this PR specifically?

Reply:
This it not needed specifically.. It was needed only because we run live server which appends that script..But leave it, I think it is ok for future. It does not have any influece on tests, though

Reply:
But why didn't it cause problems for people running the live server previously?

Reply:
But why didn't it cause problems when running the live server previously?

Reply:
It did, just no one needed to practically check last image as we didnt have failures like you did. So at first we (you and me) thought that this is the issue (that selector), but actually it was not, cause no script is embedded while running tests.